src/main/java/org/example/service/menu/UserServiceManager.java

```diff
@@ -0,0 +1,106 @@
+package org.example.service.menu;
+
+import org.example.UserContext;
+import org.example.model.User;
+import org.example.service.UserService;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.Scanner;
+
+public class UserServiceManager {
+
+    private final Scanner scanner;
+    private final UserService service;
+    private final UserContext userContext;
+
+    public UserServiceManager(UserService service, Scanner scanner) {
+        this.scanner = scanner;
+        this.service = service;
+        this.userContext = UserContext.getInstance();
+    }
+
+    public void getRegistration() {
```

Comment:
Method named like get.. should return something. So, I recommend change the name
```suggestion
    public void makeRegistration() {
```